processing/src/main/java/io/druid/query/filter/SelectorDimFilter.java

```diff
@@ -84,6 +112,18 @@ public String getValue()
     return value;
   }
 
+  @JsonProperty
+  public ExtractionFn getExtractionFn()
+  {
+    return extractionFn;
+  }
+
+  @Override
+  public String toString()
+  {
+    return String.format("%s(%s) = %s", extractionFn, dimension, value);
```

Comment:
nit: this could be prettier-printed if we just do do `dim = val` when extractionFn is null.